case_studies/dc2/full_train_config.yaml

```diff
@@ -0,0 +1,62 @@
+---
+defaults:
+    - ../../bliss/conf@_here_: base_config
+    - _self_
+    - override hydra/job_logging: stdout
+
+mode: train
+
+encoder:
+    survey_bands: ["g", "i", "r", "u", "y", "z"]
+    tile_slen: 4
+    double_detect: false
+    tiles_to_crop: 1
+    do_data_augmentation: true
+    image_normalizer:
+        bands: [0, 1, 2, 3, 4, 5]
+        num_psf_params: 4  # for SDSS, 4 for DC2
+        log_transform_stdevs: []
+        use_clahe: false
+        asinh_params:
+            scale: 0.1
+            thresholds: [-3, 0, 1, 3]
+        concat_psf_params: true
+    metrics:
+        _target_: torchmetrics.MetricCollection
+        metrics:
+          - _target_: bliss.encoder.metrics.DetectionPerformance
+            mag_bin_cutoffs: [19, 19.4, 19.8, 20.2, 20.6, 21, 21.4, 21.8]
+          - _target_: bliss.encoder.metrics.DetectionPerformanceOnFlux
+            flux_bin_cutoffs: [200, 400, 600, 800, 1000]
+          - _target_: bliss.encoder.metrics.SourceTypeAccuracy
+          - _target_: bliss.encoder.metrics.FluxError
+            survey_bands: ${encoder.survey_bands}
+          - _target_: bliss.encoder.metrics.GalaxyShapeError
+
+
+surveys:
+    dc2:
+        _target_: bliss.surveys.dc2.DC2
+        data_dir: /data/scratch/dc2local/run2.2i-dr6-v4/coadd-t3828-t3829/deepCoadd-results/
+        split_result_folder: split_results_05_16_2024
+        cat_path: ${paths.dc2}/merged_catalog/merged_catalog_with_flux_over_100.pkl
+        batch_size: 64
+        n_split: 50
```

Comment:
Could we make `n_split` much smaller (5 perhaps) and set `batch_size` to 1? Then there'd be fewer light sources missed because they are on the border between splits.

Reply:
we will address this in next PR